epilepsy12/views/organisation_views.py

```diff
@@ -427,3 +432,29 @@ def view_preference(request, organisation_id, template_name):
     return HttpResponseClientRedirect(
         reverse(template_name, kwargs={"organisation_id": organisation.pk})
     )
+
+
+@login_and_otp_required()
+@permission_required("epilepsy12.can_publish_epilepsy12_data")
+def kpi_download(request, organisation_id):
+    """
+    GET: Loads the page necessary for downloading KPIs
+    """
+
+    context = {"organisation_id": organisation_id}
+
+    template_name = "epilepsy12/partials/kpis/kpi_download.html"
+
+    return render(request=request, template_name=template_name, context=context)
+
+
+@login_and_otp_required()
+@permission_required("epilepsy12.can_publish_epilepsy12_data")
```

Comment:
[from code review] this is equivalent to the RCPCH team (superuser)

Reply:
perhaps just me but I am not sure that it is inappropriate to use this permission for this use case here. The point of E12 staff downloading the data is for the purposes of publication ultimately, and often the data analysis they will be doing here may be a precursor to formal publication, so to my mind this is part of the same thing. I am not sure what adding a new permission would gain, and it would create a new migration also.

Reply:
I agree it's acceptable for now, we just flagged the permission naming as a potential issue for the future. I think it's a stretch for most people to see that downloading data is 'eventually equivalent' to publication, because really there is no guarantee that the tweo activities are linked. Some downloaded data might be for other purposes.

If we find we are 'overloading' this permission in the future, we could consider either:
* a new permission specifically for downloading of E12 data like `can_download_epilepsy12_data`
* a broader permission, something like `is_epilepsy12_team`

At this stage fine as it is, but this is the kind of stuff that makes maintainability harder for other people less familiar with the codebase, so it's a thing to keep tabs on.